README.md

```diff
@@ -442,6 +442,40 @@ Notice a few things about our single-file component:
 
 Try running `npm run build` and open up `index.html` to see the result!
 
+# Use Decorators to define a component
+
+Components can also be defined using [Decorators](https://www.typescriptlang.org/docs/handbook/decorators.html). With the help of two additional packages ([vue-class-component](https://github.com/vuejs/vue-class-component) and [vue-property-decorator](https://github.com/kaorun343/vue-property-decorator)), the definition of our component can be simplified as following:

```

Comment:
Lowercase the `D` in decorators, use 1 line per sentence, and avoid using the word "simplified" (e.g. written as follows).